Libraries/CameraRoll/RCTCameraRollManager.m

```diff
@@ -20,6 +20,8 @@
 #import "RCTLog.h"
 #import "RCTUtils.h"
 
+#if !TARGET_OS_TV
```

Comment:
Same here. If the whole contents of a file doesn't make on Apple TV, put the gating at the file level.